docs/src/main/sphinx/release/release-443.md

```diff
@@ -0,0 +1,46 @@
+# Release 443 (21 Mar 2024)
+
+## General
+
+* Ensure consistent formatting when casting from double or real to varchar. ({issue}`21136`)
+* Prevent query failure when partial aggregation over long decimals is pushed
```

Comment:
"Long decimal" is an implementation detail. Replace with "decimals with precision larger than 18"